youtube_dl/extractor/adobepass.py

```diff
@@ -16,21 +16,6 @@
 
 
 MSO_INFO = {
-    'DTV': {
-        'name': 'DIRECTV',
-        'username_field': 'username',
-        'password_field': 'password',
```

Comment:
Aren't "username" and "password" the default anyway?  It seemed redundant to me, but if not, then I apologize for removing it.